modules/git/repo_commit_gogit.go

```diff
@@ -50,7 +50,9 @@ func (repo *Repository) ConvertToSHA1(commitID string) (SHA1, error) {
 		}
 	}
 
-	actualCommitID, err := NewCommand(repo.Ctx, "rev-parse", "--verify", commitID).RunInDir(repo.Path)
+	stdout := new(strings.Builder)
+	err := NewCommand(repo.Ctx, "rev-parse", "--verify", commitID).RunWithContext(&git.RunContext{Dir: repo.Path, Timeout: -1, Stdout: stdout})
```

Comment:
```suggestion
	err := NewCommand(repo.Ctx, "rev-parse", "--verify", commitID).RunWithContext(&RunContext{Dir: repo.Path, Timeout: -1, Stdout: stdout})
```

That should fix the build error.

Reply:
I see